resources/nexus.sh

```diff
@@ -6,67 +6,147 @@ echo "$(date) - LDAP Enabled: ${LDAP_ENABLED}"
 
 # Copy config files.
 mkdir -p ${NEXUS_HOME}conf
-cp -R /resources/* ${NEXUS_HOME}conf
+cp -R /resources/conf/managed/* ${NEXUS_HOME}conf
+cp -R -n /resources/conf/unmanaged/* ${NEXUS_HOME}conf
```

Comment:
I think it'd be useful to have a comment near this block explaining the difference between the managed and unmanaged files please.